README.md

```diff
@@ -3,49 +3,96 @@
+how all the properties relate to each other. An implementation of the data model therefore describes how a specific met mast was 
+installed, how the sensors were mounted on that met mast, how these sensors were programmed into the logger and how 
+all these properties may have changed over time.
 
-- [JSON Schema](./schema/iea43_wra_data_model.schema.json): This JSON schema describes how a JSON file that describes wind resource measurement
-  data should be composed.
+For more information on the definition of a Data Model and other terminology such as Schema, please see the 
+[Task 43 Glossary](https://iea-task-43.gitbook.io/iea-task-43-glossary/terms/data-model).
 
-- [Schema documentation](./docs/README.md): Markdown documentation for the schema.
+<br>
 
-- [Form App](https://iea-task-43.github.io/digital_wra_data_standard/): This app shows a form that is modeled after the JSON schema and can create JSON data out of your inputs
-  that is in accordance with the proposed standard.
+# The WRA Data Model
 
-- [Python Schema Loading Example](./tools/load_demo_schema.ipynb): This notebook shows how to read an example file that uses the JSON schema with Python. [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/IEA-Task-43/digital_wra_data_standard/master?filepath=.%2Ftools%2Fload_demo_schema.ipynb)
+The _WRA Data Model_ provides the instructions for how to digitally represent an installed met mast, lidar, sodar, 
+floating lidar or solar measurement station. An implementation of the data model therefore can contain the latitude and 
+longitude of where the met mast was installed. At what height is a wind speed measurement been made and by what 
+sensor. How is this sensor mounted onto the mast. How the sensor is connected to the logger and how this logger channel
+is programmed (i.e. what slope and offset values are programmed into the logger). And how all these properties 
+may have changed over time.
 
-**Pipeline Status**
-| Pipeline | Status | Result |
-|:---------|:-------|:-------|
-| Documentation | ![Compile Documentation to Markdown](https://github.com/IEA-Task-43/digital_wra_data_standard/workflows/Compile%20Documentation%20to%20Markdown/badge.svg) | [Documentation](./docs/README.md) |
-| Form App | ![Deploy Form App to GitHub Pages](https://github.com/IEA-Task-43/digital_wra_data_standard/workflows/Deploy%20Form%20App%20to%20GitHub%20Pages/badge.svg) | [Form App](https://iea-task-43.github.io/digital_wra_data_standard/) |
+To learn more about the _WRA Data Model_, please read Amit Bohara's 
+[introduction](https://github.com/IEA-Task-43/digital_wra_data_standard/wiki/Task-43-WRA-Data-Model---An-introduction)
+in the Wiki section of this GitHub repository.
 
-## Digital Calibration Certificate Model
+JSON Schema is used to capture the _WRA Data Model_. It can be thought of as a blueprint for how an implementation of the
```

Comment:
Suggest spelling out Java Script Object Notation (JSON) and maybe including a reference link for newbies